### ISSUE TITLE
Website needs introductory text

### ISSUE THREAD

**Issue by chosak** [cfpb/cfgov-lighthouse#39]
The dashboard (https://cfpb.github.io/cfgov-lighthouse/) doesn't include any introductory or overview text describing what it is for or how to use it. Additionally it could/should link back to the source code repository.

As reported by @sonnakim on #38.

**Comment by mgifford** [comment]
Agreed.  And why not just use https://github.com/GoogleChrome/lighthouse-ci/

Knowing how this is different from the upstream code is important.
